metrics/microprofile/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright DataStax, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.datastax.oss</groupId>
+    <artifactId>java-driver-parent</artifactId>
+    <version>4.8.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>java-driver-metrics-microprofile</artifactId>
+  <packaging>bundle</packaging>
+  <name>DataStax Java driver for Apache Cassandra(R) - Metrics - microprofile</name>
+  <properties>
+    <microprofile.version>2.3</microprofile.version>
```

Comment:
shouldn't it work for provided dependencies to not hardcode versions of the library in this module? (same for micro profile)

Reply:
Maven won't compile if I remove the dependency version. I can change it to a range and it will pick the latest.

Reply:
You can also use `RELEASE`, but usually people prefer a fixed version because variable versions or version ranges tend to produce non-repeatable builds.